tests/dbz-twin/rac/fuzz-test.sh

```
@@ -427,19 +428,20 @@ action_validate() {
-        logminer)   docker logs fuzz-dbz-logminer 2>&1 ;;
-        olr)        docker logs fuzz-dbz-olr 2>&1 ;;
-        consumer)   docker logs fuzz-consumer 2>&1 ;;
-        validator)  docker logs fuzz-validator 2>&1 ;;
-        olr-vm)     ssh $_SSH_OPTS "${VM_USER}@${VM_HOST}" "podman logs $OLR_CONTAINER" 2>/dev/null ;;
+        kafka)          docker logs fuzz-kafka 2>&1 ;;
+        logminer)       docker logs fuzz-dbz-logminer 2>&1 ;;
+        olr)            docker logs fuzz-dbz-olr 2>&1 ;;
+        lob-logminer)   docker logs fuzz-dbz-lob-logminer 2>&1 ;;
+        consumer)       docker logs fuzz-consumer 2>&1 ;;
+        validator)      docker logs fuzz-validator 2>&1 ;;
+        olr-vm)         ssh $_SSH_OPTS "${VM_USER}@${VM_HOST}" "podman logs $OLR_CONTAINER" 2>/dev/null ;;
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)